Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\n###> symfony/framework-bundle ###\r\n\r\n/config/secrets/prod/prod.decrypt.private.php\r\n\r\n###< symfony/framework-bundle ###\r\n\r\n###> phpunit/phpunit ###\r\n/phpunit.xml\r\n.phpunit.result.cache\r\n###< phpunit/phpunit ###\r\n/var/\r\n/migrations/\r\n###> symfony/phpunit-bridge ###\r\n###< symfony/phpunit-bridge ###\r\n\r\n###> symfony/asset-mapper ###\r\n/var/cache/\r\n/var/log/\r\n###< symfony/asset-mapper ###\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 34b44ed32d3a17fe106362edf5712dde3592780d)
+++ b/.gitignore	(date 1776948037743)
@@ -4,7 +4,7 @@
 /config/secrets/prod/prod.decrypt.private.php
 
 ###< symfony/framework-bundle ###
-
+/vendor/
 ###> phpunit/phpunit ###
 /phpunit.xml
 .phpunit.result.cache
